avm/res/key-vault/vault/README.md

```diff
@@ -1394,8 +1393,4 @@ Resource tags.
 
 ## Cross-referenced modules
 
-This section gives you an overview of all local-referenced module files (i.e., other CARML modules that are referenced in this module) and all remote-referenced files (i.e., Bicep modules that are referenced from a Bicep Registry or Template Specs).
-
-| Reference | Type |
-| :-- | :-- |
-| `br/public:avm-res-network-privateendpoint:0.1.1` | Remote reference |
+_None_
```

Comment:
Nope :bomb:

Reply:
Regen & fixed. Was an issue with the port from an improvement on the CARML-side and the differing paths over there